policies/example-policy.js

```diff
@@ -2,8 +2,15 @@ module.exports = {
   name: 'example',
   policy: (actionParams) => {
     return (req, res, next) => {
-      // eslint-disable-next-line no-console
       console.log('executing policy-from-example-plugin with params', actionParams);
     };
+  },
+  schema: {  // This is for Admin API to validate params
+    url: {
```

Comment:
I'm afraid this is not a valid schema. It should be:

```javascript
{
    type: "object",
    properties: {
        url: {
            title: 'url',
            description: 'the url to initialize',
            type: 'string',
            required: true
        }

    }
}
```

Reply:
$id is required now?

Reply:
It has always been required as far as I remember